weblate/settings.py

```diff
@@ -342,6 +318,7 @@
     'weblate.trans',
     'weblate.lang',
     'weblate.accounts',
+    'libravatar',
```

Comment:
I'm not 100% sure, (not a django expert) but I am pretty sure adding this is what got the green tick in the settings page.